test/performance/comm/low-level/many-to-one.ml-compopts

```diff
@@ -1,7 +1,7 @@
--sop=opGet      -srunSecs=5 many-to-one-helper.h # many-to-one-gets
--sop=opPut      -srunSecs=5 many-to-one-helper.h # many-to-one-puts
--sop=opFetchAMO -srunSecs=5 many-to-one-helper.h # many-to-one-fetchAmos
--sop=opAMO      -srunSecs=5 many-to-one-helper.h # many-to-one-amos
--sop=opBuffAMO  -srunSecs=5 many-to-one-helper.h # many-to-one-buffAmos
--sop=opFastOn   -srunSecs=5 many-to-one-helper.h # many-to-one-fastOns
--sop=opOn       -srunSecs=5 many-to-one-helper.h # many-to-one-ons
+-sop=opGet      -srunSecs=5.0 many-to-one-helper.h # many-to-one-gets
+-sop=opPut      -srunSecs=5.0 many-to-one-helper.h # many-to-one-puts
+-sop=opFetchAMO -srunSecs=5.0 many-to-one-helper.h # many-to-one-fetchAmos
+-sop=opAMO      -srunSecs=5.0 many-to-one-helper.h # many-to-one-amos
```

Comment:
Alternatively you could probably just update the source to say `config const runSecs:real = 5.0;`

Reply:
Have a preference?  This isn't an area where I feel I have a lot of experience making the choice.

Reply:
I would update the source, just to minimize the diff, but you already this here, so I don't have a preference